docs/troubleshooting/network-diagrams.mdx

```diff
@@ -5,19 +5,6 @@ title: "🕸️ Network Diagrams"
 
 Here, we provide clear and structured diagrams to help you understand how various components of the network interact within different setups. This documentation is designed to assist both macOS/Windows and Linux users. Each scenario is illustrated using Mermaid diagrams to show how the interactions are set up depending on the different system configurations and deployment strategies.
 
-## Table of Contents
```

Comment:
Any reason why this part has been removed?

Reply:
It was a combination of three things😊

1. It's the only page with a ToC on the whole site.
2. It contained a couple of broken links due to some headings being changed since the ToC was created.
3. Docusaurus already displays an up-to-date ToC on the right side of the page so it felt redundant.

Reply:
Sounds reasonable!